services/core-web/src/components/mine/Securities/BondDocumentsTable.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
```

Comment:
Instead of creating this new table, can we use 'DocumentTable.js' in common, It might need some tweaks to make is fully reusable, but should be used for this exact situation.

Reply:
Addressed in-person. Conclusion: Will make DocumentTable.js reusable (as it's currently Variance-specific).